### ISSUE TITLE
Update Pattern template in Figma (to have variables version) - Navigation

### ISSUE THREAD

**Issue by yunjungyeh** [jpmorganchase/salt-ds#4272]
Update Pattern template in Figma (to have variables version) - Navigation

**Comment by bhoppers2008** [comment]
No further updates to be made to retired libraries.

@yunjungyeh please review if this is still required for Salt DS Figma.

**Comment by yunjungyeh** [comment]
Yes as we are still linking the Pattern template file from Salt DS library

**Comment by bhoppers2008** [comment]
On hold pending new navigation work from @dplsek
